### ISSUE TITLE
ZLS doesn't compile with latest zig

### ISSUE THREAD

**Issue by namnd** [zigtools/zls#960]
### Zig Version

0.11.0-dev.1557+03cdb4fb5

### Zig Language Server Version

7819d918a04867f82981f3ef849e61bce737f400

### Steps to Reproduce

$ zig build -Drelease-safe

### Expected Behavior

Compiled

### Actual Behavior

```
me@namnd $ zig build -Drelease-safe                                                                                                                                       (13.27s) master
/home/me/zig/zls/build.zig:17:19: error: no field or member function named 'standardReleaseOptions' in 'Build'
    const mode = b.standardReleaseOptions();
                 ~^~~~~~~~~~~~~~~~~~~~~~~
/home/me/zig/build/stage3/lib/zig/std/Build.zig:1:1: note: struct declared here
const std = @import("std.zig");
^~~~~
referenced by:
    runBuild__anon_5456: /home/me/zig/build/stage3/lib/zig/std/Build.zig:1526:37
    main: /home/me/zig/build/stage3/lib/zig/build_runner.zig:214:16
    remaining reference traces hidden; use '-freference-trace' to see all reference traces

```

**Comment by leecannon** [comment]
will be fixed by #955

**Comment by namnd** [comment]
Verify it's working now
